storageStruct.go

```diff
@@ -67,7 +67,12 @@ type ServerST struct {
 	HTTPSKey           string       `json:"https_key" groups:"api,config"`
 	HTTPSAutoTLSEnable bool         `json:"https_auto_tls" groups:"api,config"`
 	HTTPSAutoTLSName   string       `json:"https_auto_tls_name" groups:"api,config"`
+	ICEServers         []string     `json:"ice_servers" groups:"api,config"`
```

Comment:
How about updating the `README.md` also?

Reply:
Happy with this in the latest commit?

Reply:
Yes, looks good, thanks.